pages/who-we-are/board.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: layouts/meet-the-board.njk
+title: Who we are
+date: 2017-01-01T00:00:00.000Z
+permalink: /who-we-are/board.html
```

Comment:
@SamMed1 I noticed that some pages include the .html file type. It would be idea if we could remove them. Do you know of a good reason for keeping them?